app/src/main/java/ch/epfl/sweng/studyup/course/Course.java

```diff
@@ -0,0 +1,24 @@
+package ch.epfl.sweng.studyup.course;
+
+import java.util.UUID;
+
+public class Course {
```

Comment:
Why is this not an enum? It would make much more sense, courses are predefined and unique.